### PR TITLE
Add-move-member-to-team

### PR DESCRIPTION
**Pull request by HoangVanTrinh92**

### CODE REVIEW

File: app/models/team.rb [hunk 1]
```diff
@@ -1,2 +1,5 @@
 class Team < ActiveRecord::Base
+  has_many :users, through: :team_users
+  has_many :team_users
+  accepts_nested_attributes_for :team_users, allow_destroy: true
```
Comment (phonghenry14) [inline app/models/team.rb]: cách dòng trên ra nhé

File: app/views/admin/team_users/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<%= form_for([:admin, @team]) do |f| %>
```
Comment (phonghenry14) [inline app/views/admin/team_users/_form.html.erb]: bỏ ()

File: app/views/admin/team_users/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        <%= f.collection_check_boxes :user_ids, @users, :id, :email do |b| %>
+          <tr>
+            <%= b.label do %>
+              <th><%= b.text %></th>
+              <th class="selection_checkbox"><%= b.check_box %></th>
+            <% end %>
+          </tr>
+        <% end %>
+      </table>
+      <%= f.submit "Save", class: "btn btn-primary" %>
```
Comment (nghiabk) [inline app/views/admin/team_users/_form.html.erb]: sao ko cho cái Save vào I18n

File: app/views/admin/team_users/_team_user.html.erb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<%= div_for team_user do %>
+  <strong><p class="team_user-user-email"><%= sanitize(team_user.user.email) %></p></strong>
+  <span class="date">
+    <small><%= time_ago_in_words(team_user.created_at) %> ago</small>
```
Comment (nghiabk) [inline app/views/admin/team_users/_team_user.html.erb]: remove ()

File: app/views/admin/team_users/create.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<% if @team_user.errors.any? %>
+  $('.team_user-error').html('<%= escape_javascript(render 'shared/error_messages', obj: @team_user) %>');
```
Comment (nghiabk) [inline app/views/admin/team_users/create.js]: Use "" instead of ''

File: app/views/admin/team_users/show.html.erb [hunk 6]
```diff
@@ -0,0 +1,23 @@
+<h1><%= t "team.add_member" %></h1>
+<div id="assign-form-content">
+  <div class="list-user">  
+    <h2><%= t "team.list_member"%></h2>
+    <%= render "form" %>
+  </div>
+  <div class="list-user-assigned">
+    <h2><%= t "team.list_member_assigned"%></h2>
+    <div class="team_users">
+      <h3 id="team_users-count"><%= t("team.has") %> <%= pluralize(@team.team_users.count, "team_user") %></h3>
```
Comment (nghiabk) [inline app/views/admin/team_users/show.html.erb]: remove ( )

File: app/views/admin/teams/show.html.erb [hunk 7]
```diff
@@ -17,3 +17,16 @@
 
 <%= link_to t("user.edit"), edit_admin_team_path(@team) %> |
 <%= link_to t("user.back"), admin_teams_path %>
+
+<h1><%= t "team.list_member" %></h1>
+<table>
+  <tr><th><%= t "team.user_name" %></th></tr>
+    <% @users.each do |user| %>
+    <tr>
```
Comment (nghiabk) [inline app/views/admin/teams/show.html.erb]: indent

File: app/views/admin/team_users/create.js [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+<% else %>
+  var team_users_list = $('#team_users-list');
+  team_users_list.prepend('<%= escape_javascript(render partial: @team_user) %>');
+  $('#team_users-count').html('This team has <%= escape_javascript(pluralize(@team.team_users.count, 'team_user')) %>');
```
Comment (nghiabk) [inline app/views/admin/team_users/create.js]: '#team_users-count'   -> "#team_users-count"

File: app/views/admin/teams/show.html.erb [hunk 9]
```diff
@@ -17,3 +17,16 @@
 
 <%= link_to t("user.edit"), edit_admin_team_path(@team) %> |
 <%= link_to t("user.back"), admin_teams_path %>
+
+<h1><%= t "team.list_member" %></h1>
+<table>
+  <tr><th><%= t "team.user_name" %></th></tr>
+    <% @users.each do |user| %>
```
Comment (nghiabk) [inline app/views/admin/teams/show.html.erb]: indent

File: app/assets/stylesheets/custom.css.scss [hunk 10]
```diff
@@ -293,4 +293,25 @@ aside {
...
+  width: 800px;
+  float: left;
+}
+
+.button1{
+  width: 1200px;
+  height: 300px;
+  margin: 10px;
+  padding: 10px;
 }
```
Comment (khanhkhanh) [inline app/assets/stylesheets/custom.css.scss]: Break line

File: app/assets/stylesheets/custom.css.scss [hunk 11]
```diff
@@ -293,4 +293,25 @@ aside {
...
+}
+
+.list-user-assigned{
+  background-color: #9D9D9D;
+  border: 1px solid #bbb;
+  width: 800px;
+  float: left;
+}
+
+.button1{
```
Comment (khanhkhanh) [inline app/assets/stylesheets/custom.css.scss]: Need a space before {

File: app/assets/stylesheets/custom.css.scss [hunk 12]
```diff
@@ -293,4 +293,25 @@ aside {
...
+}
+
+.list-user{
+  background-color: #9D9D9D;
+  border: 1px solid #bbb;
+  width: 310px;
+  float: right;
+}
+
+.list-user-assigned{
```
Comment (khanhkhanh) [inline app/assets/stylesheets/custom.css.scss]: Need a space before {

File: app/assets/stylesheets/custom.css.scss [hunk 13]
```diff
@@ -293,4 +293,25 @@ aside {
 
 .th-colspan{
   width: 270px;
+}
+
+.list-user{
```
Comment (khanhkhanh) [inline app/assets/stylesheets/custom.css.scss]: Need a space before {

File: app/assets/stylesheets/custom.css.scss [hunk 14]
```diff
@@ -293,4 +293,25 @@ aside {
 
 .th-colspan{
```
Comment (khanhkhanh) [inline app/assets/stylesheets/custom.css.scss]: Need a space before {

File: app/views/layouts/_header.html.erb [hunk 15]
```diff
@@ -1,3 +1,5 @@
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
```
Comment (khanhkhanh) [inline app/views/layouts/_header.html.erb]: No need this line. You have required jquery in application.js

File: app/views/layouts/_header.html.erb [hunk 16]
```diff
@@ -1,3 +1,5 @@
...
+
 <header class="navbar navbar-fixed-top navbar-inverse">
   <div class="container">
     <%= link_to "MMS_8", root_path, id: "logo" %>
```
Comment (khanhkhanh) [inline app/views/layouts/_header.html.erb]: i18n

File: config/locales/en.yml [hunk 17]
```diff
@@ -88,6 +88,25 @@ en:
     listing_position: "Listing positions"
```
Comment (khanhkhanh) [inline config/locales/en.yml]: ??? You meant "List of positions" ???

File: config/locales/en.yml [hunk 18]
```diff
@@ -104,4 +123,10 @@ devise:
     name: "Name"
     description: "Description"
     leader: "Leader"
+    add_member: "Add member to Team"
+    view_detail: "View Team details"
+    back_to_team_manager: "Back to Team manager"
+    list_member: "List members is attend this Team"
+    user_name: "User Name"
+    save: "Save"
```
Comment (khanhkhanh) [inline config/locales/en.yml]: Break line

**Pull request merged**